.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy-release:
+    name: Deploy Release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Github CLI
+        run: build-tools/release/install-hub.sh ${{ matrix.os }}
+        shell: bash
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - name: Build SeaORM CLI
+        run: build-tools/release/build-cli.sh ${{ matrix.os }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        shell: bash
+  publish-crate:
+    name: Publish crate
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - name: Publish crate
+        run: build-tools/release/publish-crate.sh
+        env:
+          CRATES_TOKEN: ${{ secrets.CRATES_TOKEN }}
```

Comment:
Hey @tristan-zander, sorry for the delay. All in all, it looks good!

I think we better publish the crate manually instead of relying on GitHub Actions. Could you please remove this part from the workflow and `build-tools/release/publish-crate.sh` file.